views/stylesheet.etlua

```diff
@@ -77,14 +77,18 @@
     color: darkgreen;
     text-decoration: none;
   }
-      
+
   /*IE6 fix*/
   * html body{
     padding: 100px 0 0 230px; /* Set the first value to the height of the header and last value to the width of the nav */
   }
 
-  * html main{ 
-    height: 100%; 
-    width: 100%; 
+  * html main{
+    height: 100%;
+    width: 100%;
+  }
+
+  .form-control {
```

Comment:
This is the only change here, the rest is trimming whitespace at the end of lines.